templates/base/head.tmpl

```diff
@@ -42,6 +42,7 @@
 				MaxTimeout: {{NotificationSettings.MaxTimeout}},
 				EventSourceUpdateTime: {{NotificationSettings.EventSourceUpdateTime}},
 			},
+			EnableTimetracking: {{if EnableTimetracking}}true{{else}}false{{end}},
```

Comment:
as a side note, we could probably get these into the system info api endpoint to better integrate SPAs in the future (this isn't a note for this PR though).

Reply:
same with lfs enable/disable info ... - will add a pull

Reply:
https://github.com/go-gitea/gitea/pull/14753

Reply:
awesome. thanks :)